### ISSUE TITLE
Path shows the concatenated value on the validation result

### ISSUE THREAD

**Issue by sezgigumus** [daveshanley/vacuum#276]
If the invalid path is located in the second place of the paths or late, the results show concatenated path. Please see the example.

**ruleset.yaml:**
```
extends: [["spectral:oas", off]]
rules:
  version-in-path:
        description: "Version needs to be present in the path as this is required for analytics tracking"
        formats:
            - oas3
            - oas3_1
        recommended: true
        given: $.paths
        howToFix: add version number to the path
        id: version-in-path
        resolved: true
        severity: error
        then:
            function: pattern
            functionOptions:
                match: .*\/[vV]\d+\/.*$
        type: validation
```

**routes.yaml**
```
openapi: 3.0.0

info:
  version: latest
  title: Load testing APIs
  description: These APIs will be used to load test
servers:
  - url: https://services.test.com
    description: Production
paths:
  /api/testing/v1/organizations:
    get:
      operationId: get-organizations
      summary: get-organizations
      description: get-organizations
      responses:
        "200":
          description: ok
    post:
      operationId: post-organizations
      summary: post-organizations
      description: post-organizations
      responses:
        "200":
          description: ok
  /api/testing/organizations/{organizationId}:
    parameters:
      - in: path
        name: organizationId
        required: true
        schema:
          type: string
    get:
      operationId: get-organization
      summary: get-organization
      description: get-organization
      responses:
        "200":
          description: ok
```

running this command on `v.0.0.55`
`vacuum report -r ruleset.yaml routes.yaml `

Results:
```
            {
                "message": "Version needs to be present in the path as this is required for analytics tracking: `/api/testing/organizations/{organizationId}` does not match the expression `.*\\/[vV]\\d+\\/.*$`",
                "range": {
                    "start": {
                        "line": 26,
                        "character": 3
                    },
                    "end": {
                        "line": 26,
                        "character": 3
                    }
                },
                "path": "$.paths./api/testing/v1/organizations./api/testing/organizations/{organizationId}./api/testing/organizations/{organizationId}",
                "ruleId": "version-in-path",
                "ruleSeverity": "error"
            }
````

`$.paths./api/testing/v1/organizations./api/testing/organizations/{organizationId}./api/testing/organizations/{organizationId}` is not clear and makes user confusing. Can we only show the invalid path?

**Comment by daveshanley** [comment]
Oh, yes, this looks confusing. I will look into this and see what can be done to clean it up.

**Comment by daveshanley** [comment]
This has been fixed in `v0.1.6`. Paths are cleanly extracted, and now things are clear.
